test/threads.jl

```diff
@@ -96,9 +98,10 @@ end
 const AFFINITY_SUPPORTED = (Sys.islinux() || Sys.iswindows()) && !running_under_rr()
```

Comment:
You can potentially drop the rr check, since it will be implied (it sets an affinity mask to 1 cpu)

Reply:
Let's see how it goes: https://github.com/JuliaLang/julia/pull/44693